pkgs/development/python-modules/Theano/theano-with-cuda/default.nix

```diff
@@ -56,7 +56,7 @@ buildPythonPackage rec {
     pycuda
     cudatoolkit
     libgpuarray
-  ] ++ (stdenv.lib.optional (cudnn != null) [ cudnn ]);
+  ] ++ (stdenv.lib.optional (cudnn != null) cudnn);
```

Comment:
Why would cudnn be optional? I think there's no reason and it should simply be added to the list, unconditionally.